Server.js

```diff
@@ -50,7 +20,40 @@ function IsJSON(data)
   return true
 }
 
+function GetFullTime(){
+  var date_obj = new Date()
+  var day = date_obj.getDate()
+  var month = date_obj.getMonth() + 1
+  var year = date_obj.getFullYear()
+  var time = date_obj.getHours() + ":" + date_obj.getMinutes() + ":" + date_obj.getSeconds()
+
+  return day + "-" + month + "-" + year + ", " + time
+}
+
+app.get('/api/tasks/:id', function(req, res){
```

Comment:
please put all the tasks api and logic in a separate tasksRoutes.js file under routes folder